client/web/compose/src/components/PageBlocks/MetricConfigurator/index.vue

```diff
@@ -262,23 +262,38 @@
               >
                 <template #label>
                   {{ $t('metric.drillDown.label') }}
+
                   <b-form-checkbox
                     v-model="edit.drillDown.enabled"
                     switch
-                    class="ml-1"
+                    class="ml-1 mb-1"
                   />
                 </template>
 
-                <c-input-select
-                  v-model="edit.drillDown.blockID"
-                  :options="drillDownOptions"
-                  :disabled="!edit.drillDown.enabled"
-                  :get-option-label="o => o.title || o.kind"
-                  :reduce="option => option.blockID"
-                  :clearable="true"
-                  :placeholder="$t('metric.drillDown.openInModal')"
-                  append-to-body
-                />
+                <b-input-group>
+                  <c-input-select
+                    v-model="edit.drillDown.blockID"
+                    :options="drillDownOptions"
+                    :disabled="!edit.drillDown.enabled"
+                    :get-option-label="o => o.title || o.kind"
+                    :reduce="option => option.blockID"
+                    :clearable="true"
+                    :placeholder="$t('metric.drillDown.openInModal')"
+                    append-to-body
+                  />
+
+                  <b-input-group-append>
+                    <column-picker
+                      :module="recordListModule"
+                      :disabled="!!edit.drillDown.blockID || !edit.drillDown.enabled"
+                      :fields="selectedFields"
```

Comment:
when column picker is used in a b-input-group, the button variant should be 'extra-light'